javascriptlib/src/mill/javascriptlib/ReactScriptsModule.scala

```diff
@@ -0,0 +1,174 @@
+package mill.javascriptlib
+
+import mill._
+import os.*
+
+// create-react-app scripts
+trait ReactScriptsModule extends TypeScriptModule {
```

Comment:
If this is specific to `create-react-app`, should it be `CreateReactAppModule`?

Reply:
Will update the name

Reply:
I changed it back to `ReactScriptsModule` cra runs with react-scripts which is a package different from React